### PR TITLE
Feature/pomodoro clock

### PR DESCRIPTION
**Pull request by nirmit-sawhney**
Completed the 1st challenge i.e. **Pomodoro Timer**.

### CODE REVIEW

File: pomodoro_clock/script.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+function disableInp(){
+    min.disabled=true;
+    sec.disabled=true;
+}
+
+start.addEventListener('click',()=>{
+    disableInp();
+    if(start.innerHTML=="start"){
+        if(!(min.value>=0 && min.value<=99) || !(sec.value>=0 && sec.value<=60) || min.value=="" || sec.value=="" || (min.value==0 && sec.value==0)){
```
Comment (vrknetha) [inline pomodoro_clock/script.js]: convert it to a function which returns a boolean

File: pomodoro_clock/script.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            ring.style.stroke="#09A65A";
+        }else{
+            startTimer();
+        }
+    }else{
+        stopTimer();
+    }
+})
+
+function startTimer(){
```
Comment (vrknetha) [inline pomodoro_clock/script.js]: use arrow functions

File: pomodoro_clock/script.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                sec.value=60;
+            }else{
+                sec.value-=1;
+                sec.value=('0'+sec.value).slice(-2);
+            }
+        }
+    },1000)
+}
+
+function stopTimer(){
```
Comment (vrknetha) [inline pomodoro_clock/script.js]: create a seperate file with utility name and move functions there

**Review by vrknetha — CHANGES_REQUESTED** [review]

File: pomodoro_clock/script.js (hunk 3 above)
Comment (nirmit-sawhney) [reply pomodoro_clock/script.js]: @vrknetha I had done the required changes, kindly have a look on that and please let me know if anything else is needed.

Comment (nirmit-sawhney) [reply pomodoro_clock/script.js]: @vrknetha

**Review by vrknetha — APPROVED** [review]

**Review by vrknetha — APPROVED** [review]

**Pull request merged**